modules/serverless-kn-cli-plugins.adoc

```diff
@@ -0,0 +1,11 @@
+// Module is included in the following assemblies:
+//
+// serverless/installing-kn.adoc
+
+[id="serverless-kn-plugins_{context}"]
+= `kn` plugins
+
+The Red Hat distribution of the `kn` command contains extensions of core `kn`, called plugins. Their main use is extending `kn` to use additional commands provided on {ServerlessProductName}, for example, the Kafka stack API. Main `kn` functionality and `kn` plugins are used in the same way.
+
+Currently, Red Hat provides the `kn kafka` plugin as a Technology Preview.
```

Comment:
```suggestion
Currently, Red Hat provides the `kn source kafka` plugin as a Technology Preview.
```

Reply:
the plugin it self is called `kn-source-kafka`, the command to call it is `kn source kafka` (without hypens)